### ISSUE TITLE
Warning: Truncated 'tL,\xd7' data

### ISSUE THREAD

**Issue by tongo** [exiftool/exiftool#103]
Hi, I'm trying to export GPS data from Insta 360 One R 360 video.
The command can't export all GPS data and return this error message:
`Warning: Truncated 'tL,\xd7' data - ./VID_20211022_152729_00_004.insv`

I have done a quick look on source code to understand why this error is generated, but I cannot find the right place where is generated this error.
Why this happens?
There is a way to export all data from the video?

I'm using exiftool version 12.35.

My command settings:
`exiftool -config ./_ExifTool_Config.cfg -ee -m -p ./gpx.fmt ./VID_20211022_152729_00_004.insv > output.gpx`

My _ExifTool_Config.cfg:
[_ExifTool_Config_cfg.txt](https://github.com/exiftool/exiftool/files/7523062/_ExifTool_Config_cfg.txt)

Thanks in advance

**Comment by boardhead** [comment]
If you can share the video I can take a look to see what is happening.  If you don't want to make it public, you can email me directly: philharvey66 at gmail.com

- Phil

**Comment by tongo** [comment]
Hi Phil, thanks for your response.
I shared with you a Google Drive folder with 3 videos.
Also the other video return this warning, but with a different value:
`Warning: Truncated 'xxx' data `

Thanks in advance

**Comment by boardhead** [comment]
I downloaded the first video, and ExifTool extracts the GPS from it, but you do need to use the LargeFileSupport API option

exiftool -api largefilesupport -ee -G3 VID_20210710_152906_00_003.insv

**Comment by boardhead** [comment]
I notice you do set LargeFileSupport in your config file.  This config file works for me with the command you send and produces the attached output:

[output.gpx.txt](https://github.com/exiftool/exiftool/files/7528849/output.gpx.txt)

**Comment by tongo** [comment]
Many thanks for your time!
The output.gpx.txt for the video you tried contain all 9 min of GPS data.

Thanks

**Comment by boardhead** [comment]
So I'm confused as to why you raised the issue in the first place if your original command worked?

**Comment by tongo** [comment]
Hi Phil, the video I want to extract GPS data is 'VID_20211022_152729_00_004.insv'.
With my configuration, my command, return that warning and there was only few GPS data in output, not for all video duration, so I supposed that "Truncated data" was related to GPS data.

Then I tried your command posted here and the output was the same: few GPS data, but without the warning.
Unfortunately there is no possibility to check the GPS data on the Insta360 Desktop app, so I used the Insta 360 mobile app and I discovered that there are only few GPS data in the original video!
So the issue is not related on exiftool, but is related on my video recording procedure.
